packages/oc-server-compiler/lib/oc-webpack/lib/configurator/index.js

```diff
@@ -80,11 +61,42 @@ module.exports = function webpackConfigGenerator(options) {
         {
           test: /\.js$/,
           exclude: /node_modules/,
-          use: jsLoaders
+          use: [
+            isEnvProduction && {
+              loader: require.resolve('infinite-loop-loader')
+            },
+            {
+              loader: require.resolve('babel-loader'),
+              options: {
+                cacheDirectory: true,
+                retainLines: true,
+                sourceMaps,
+                sourceRoot: path.join(options.serverPath, '..'),
+                babelrc: false,
+                presets: [
+                  [
+                    require.resolve('babel-preset-env'),
+                    {
+                      modules: false,
+                      targets: {
+                        node: 12
+                      }
+                    }
+                  ]
+                ]
+              }
+            }
+          ].filter(Boolean)
```

Comment:
can we do !== false?

Reply:
This is pretty standard though, (Create React App does the same internally)

Reply:
fair enough, disregard the other comment then